test/nano_cfg.py

```diff
@@ -21,9 +25,16 @@
 
 process.out = cms.OutputModule("NanoAODOutputModule",
     fileName = cms.untracked.string('nano.root'),
-    outputCommands = cms.untracked.vstring("drop *", "keep *_*Table_*_*","keep edmTriggerResults_*_*_*"),
+    outputCommands = cms.untracked.vstring("drop *", "keep FlatTable_*Table_*_*","keep edmTriggerResults_*_*_*"),
```

Comment:
this way you drop the x run weights (they're not FlatTable)

Reply:
should we whitelist the types we can handle downstream?
FlatTables
TriggerResults
MergableCounterTable

that's it?